core/trino-main/src/main/java/io/trino/operator/WorkProcessorPipelineSourceOperator.java

```diff
@@ -233,15 +233,13 @@ private void workProcessorOperatorStateMonitor(WorkProcessor.ProcessState<Page>
 
             long deltaReadTimeNanos = deltaAndSet(context.readTimeNanos, sourceOperator.getReadTime().roundTo(NANOSECONDS));
 
-            long deltaDynamicFilterSplitsProcessed = deltaAndSet(context.dynamicFilterSplitsProcessed, sourceOperator.getDynamicFilterSplitsProcessed());
+            deltaAndSet(context.dynamicFilterSplitsProcessed, sourceOperator.getDynamicFilterSplitsProcessed());
```

Comment:
Why not just `context.dynamicFilterSplitsProcessed.set(sourceOperator.getDynamicFilterSplitsProcessed()`